fixtures/dom/src/components/fixtures/shadow-dom/index.js

```diff
@@ -0,0 +1,67 @@
+import FixtureSet from '../../FixtureSet';
+import TestCase from '../../TestCase';
+
+const SUPPORTS_SHADOW_DOM = 'attachShadow' in HTMLElement.prototype;
+
+const React = window.React;
+const ReactDOM = window.ReactDOM;
+
+class SelectFixture extends React.Component {
+  render() {
+    if (!SUPPORTS_SHADOW_DOM) {
+      return (
+        <div>Browser does not support Shadow DOM, no tests to execute.</div>
+      );
+    }
+
+    return (
+      <FixtureSet title="Shadow DOM" description="">
+        <TestCase title="Event listeners in shadow-dom" relatedIssues="4963">
```

Comment:
Does this test case fail without this PR? My understanding is the event would still fire, but the `target` would be wrong, the TestCase doesn't assert anything about the target, only that the event was seen.